lib/irb/init.rb

```diff
@@ -45,7 +45,7 @@ def IRB.init_config(ap_path)
 
     @CONF[:USE_SINGLELINE] = false unless defined?(ReadlineInputMethod)
     @CONF[:USE_COLORIZE] = (nc = ENV['NO_COLOR']).nil? || nc.empty?
-    @CONF[:USE_AUTOCOMPLETE] = true
+    @CONF[:USE_AUTOCOMPLETE] = !(ENV.fetch("IRB_USE_AUTOCOMPLETE", "true") == "false")
```

Comment:
One last very trivial comment since I got confused when I was about to push the merge button, I wonder why it's not `ENV.fetch("IRB_USE_AUTOCOMPLETE", "true") != "false"` or `(ENV.fetch("IRB_USE_AUTOCOMPLETE", "true") != "false")`. You generally like `!(a == b)` than `a != b`?

Reply:
No it's not any special preference. I've updated 👍